trust/list.c

```diff
@@ -253,6 +253,7 @@ p11_trust_list (int argc,
 	if (!p11_enumerate_ready (&ex, "trust-policy"))
 		exit (1);
 
+	ex.flags |= P11_ENUMERATE_CORRELATE;
```

Comment:
Some tests are failing due to this behavior change:
https://travis-ci.org/p11-glue/p11-kit/jobs/196340100
Good to amend this commit with something like ueno/p11-kit@1d9bf793dd2e22853a62988dd736693beae70a77.